ui/tests/acceptance/job-definition-test.js

```diff
@@ -8,6 +8,556 @@ import Definition from 'nomad-ui/tests/pages/jobs/job/definition';
+                IP: '12.228.2.247',
+                MBits: 10,
+                Mode: 'bridge',
+                ReservedPorts: [],
+                DynamicPorts: [],
+              },
+              {
+                Device: 'eth1',
+                CIDR: '',
+                IP: '71.92.163.164',
+                MBits: 10,
+                Mode: 'bridge',
+                ReservedPorts: [],
+                DynamicPorts: [],
+              },
+            ],
+            Ports: [
+              {
+                Label: 'pixel',
+                Value: 32161,
+                To: 25929,
+                HostIP: 'b778:a055:4422:e3ca:fd2d:e5e8:464d:8f2b',
+              },
+            ],
+          },
+          Resources: {
+            CPU: 2000,
+            MemoryMB: 1024,
+            MemoryMaxMB: 0,
+            DiskMB: 0,
+          },
+          Driver: 'qemu',
+          Name: 'task-microchip-3',
+          JobID: '',
+          VolumeMounts: [
+            {
+              Volume: 'mazie',
+              Destination: '/Vella.OReilly/dudley/#4b403a',
+              PropagationMode: '',
+              ReadOnly: true,
+            },
+            {
+              Volume: 'leora',
+              Destination: '/Tressa_Brown/ian/#6d666c',
+              PropagationMode: '',
+              ReadOnly: false,
+            },
+          ],
+          GroupNames: [],
+          WithServices: false,
+          CreateRecommendations: true,
+          ID: '4',
+          Services: [],
+        },
+      ],
+    },
+  ],
+  JobSummary: {
+    GroupNames: ['pixel-g-0', 'protocol-g-1'],
+    Summary: {
+      'pixel-g-0': {
+        Queued: 10,
+        Complete: 10,
+        Failed: 6,
+        Running: 7,
+        Starting: 4,
+        Lost: 8,
+        Unknown: 1,
+      },
+      'protocol-g-1': {
+        Queued: 4,
+        Complete: 7,
+        Failed: 2,
+        Running: 0,
+        Starting: 7,
+        Lost: 3,
+        Unknown: 5,
+      },
+    },
+    Namespace: 'namespace-1',
+    ID: '1',
+    JobID: 'hdd-panel-0',
+  },
+  Specification: {
+    Definition:
+      'job "docs" {\n  namespace = "madness"\n  group "example" {\n    task "server" {\n      service {\n        tags = ["leader", "mysql"]\n\n        port = "db"\n\n        meta {\n          meta = "for your service"\n        }\n\n        check {\n          type     = "tcp"\n          port     = "db"\n          interval = "10s"\n          timeout  = "2s"\n        }\n\n        check {\n          type     = "script"\n          name     = "check_table"\n          command  = "/usr/local/bin/check_mysql_table_status"\n          args     = ["--verbose"]\n          interval = "60s"\n          timeout  = "5s"\n\n          check_restart {\n            limit = 3\n            grace = "90s"\n            ignore_warnings = false\n          }\n        }\n      }\n    }\n  }\n}\n',
+    Type: 'hcl',
+    Variables: {
+      datacenters: ['west'],
+      external_port: 4000,
+    },
+  },
+};
+
```

Comment:
Q: why not use a factory to accomplish this? They're extensible and most things you have listed here are either already job defaults or else unrelated to the acceptance test below.

Reply:
I'm not in the camp of factory or writing test logic. It becomes painful to maintain and that's personally the part of the test suite I hate the most. If you want it, I'm happy to create it. But it's just more code to maintain.

Reply:
There's a few things here:

1. This data isn't going to be modeled because this is a raw request. We're explicitly using the API response to visualize the response itself in code mirror.
2. We could overcome this using the `afterCreate` hook. But we already have 200 lines of logic in that hook and I simply don't want to touch it.

The best solution would be to just move this into a utility file if you wish to separate this fixture from the test code. And I can address that in a follow up PR.